typhos/display.py

```diff
@@ -1437,7 +1517,7 @@ def from_class(cls, klass, *, template=None, macros=None, **kwargs):
             obj = pcdsutils.utils.get_instance_by_name(klass, **kwargs)
         except Exception:
             logger.exception('Failed to generate TyphosDeviceDisplay from '
-                             'device %s', obj)
+                             'class %s', klass)
```

Comment:
Seemingly minor fix, but catches `NameError` failure - nice!

Reply:
VSCode pointed it out to me, this must have been lurking in the code for a while

Reply:
I think this is an easy one to miss without a static analysis tool since the `obj` name is right there so it seems ok to use when it isn't